ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_node_type.schema.yml

```diff
@@ -225,6 +225,16 @@ $defs:
               Set point-to-Point interface speed for the uplink switch interface only.
               Speed should be set in the format `<interface_speed>` or `forced <interface_speed>` or `auto <interface_speed>`.
             type: str
+          uplink_mtu:
+            documentation_options:
+              table: node-type-uplink-configuration
+            description: Point-to-Point uplinks MTU in bytes. This configuration over-rides `p2p_uplinks_mtu` configuration.
+            type: int
+            convert_types:
+              - str
+            min: 68
+            max: 65535
+            default: 9214
```

Comment:
There is no default for this one. The default is to fall back to p2p_uplinks_mtu, which has a default value.
```suggestion
```

Reply:
Fixed.